app/controllers/schools/alert_reports_controller.rb

```diff
@@ -5,11 +5,11 @@ class Schools::AlertReportsController < ApplicationController
 
   def index
     authorize! :read, AlertType
+    @alerts = @school.alerts
```

Comment:
This needs to change - looking at frequency and also the *latest* rather than all

Reply:
If we don't upsert, we need the latest ones - that's ok

Reply:
Also need to use the date here!